src/request/src/middleware/src/storage/index.js

```diff
@@ -32,7 +32,7 @@ export default class Storage {
       .then(() => MemoryAdapter.load(this.name))
       .then((adapter) => {
         if (isDefined(adapter) === false) {
-          throw new Error('Unable to load a storage adapter.');
+          throw new KinveyError('Unable to load a storage adapter.');
```

Comment:
Why "throw" instead of Promise.reject()? I didn't see this mentioned in the style guide, but I think it's worth deciding on one and using only it. I vote for Promise.reject(), since it's what one is trying to do when throwing inside a promise - reject the promise, not really throw an error, the rejection is essentially a side effect of the throwing. Seems semantically different to me. Also if you accidentally throw when you are not in a promise context, it would actually throw, so that's one more thing to think about.